screens/Profile.js

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import {
+  StyleSheet,
+  View,
+  Text,
+  Image,
+  Button,
+  ScrollView,
+  TouchableOpacity,
+  Share
+} from "react-native";
+import * as WebBrowser from "expo-web-browser";
+import Header from "../components/header";
+
+class UserProfile extends React.Component {
+  static navigationOptions = {
+    header: null
+  };
+
+  viewProfile = async link => {
+    let result = await WebBrowser.openBrowserAsync(link);
+  };
+
+  onShare = async (link, name) => {
+    try {
+      const result = await Share.share({
+        message: `Checkout ${name} GitHub profile: \n\n${link}`
+      });
+      if (result.action === Share.sharedAction) {
```

Comment:
Consider refactoring this statement or removing if it is not needed.

Reply:
I used it to pre-fill a text box when one wants to share via text.

Reply:
Why are the if statements empty?

Reply:
Oh.. I had thought of specifying a response according to the action but it is not necessary so I have done away with it.